src/microbe_stage/Microbe.cs

```diff
@@ -371,6 +377,13 @@ public override void _Ready()
             SetMembraneFromSpecies();
         }
 
+        DebugPanel.Instance.Connect(nameof(DebugPanel.ToggleEntityLabel), this, nameof(OnToggleEntityLabel));
+        nameLabel = new Label();
+        AddChild(nameLabel);
+        SetInternalName();
+        nameLabel.Text = internalName;
+        nameLabel.Visible = DebugPanel.ShowEntityLabel;
```

Comment:
I prefer handling these in a single autoloaded class so that other entities like floating chunks, etc can also easily have labels on them. Doing these separately here for each entities however can lead to a hard to maintain code.

Reply:
I agree with this.